host/hackrf-tools/src/hackrf_transfer.c

```diff
@@ -1146,8 +1149,8 @@ int main(int argc, char** argv) {
 			    // This is only an approximate measure, to assist getting receive levels right:
 			    double	full_scale_ratio = ((double)stream_amplitude_now / (byte_count_now ? byte_count_now : 1))/128;
 			    double	dB_full_scale_ratio = 10*log10(full_scale_ratio);
-			    if (dB_full_scale_ratio > 1)
-			    	dB_full_scale_ratio = NAN;	// Guard against ridiculous reports
+					if (dB_full_scale_ratio > 1 || dB_full_scale_ratio == -INFINITY) // Guard against ridiculous reports
+						dB_full_scale_ratio = -0.0;
```

Comment:
Is there a reason you added the `dB_full_scale_ratio == -INFINITY` condition that is still relevant now that your other changes are applied? There is a fairly common test case of `hackrf_transfer -t /dev/zero` which this now reports as full scale instead of "-inf dBfs" which I think is more appropriate.

I like the change to `-0.0`, but I would like to see the `-INFINITY` condition removed.

Reply:
My HackRF One reports -inf when reading from /dev/zero. Am I understanding correctly that this is the intended behavior?
![image](https://user-images.githubusercontent.com/6304827/182495884-487207c3-0bb5-49d7-a5d5-1bf8ec57745b.png)

I made the suggested change in my last commit.

Reply:
Yes, that is the intended behavior. Decibels are weird in this case because they describe a ratio where there is a zero in the numerator. Maximum power is 0 dBfs. Half power is -3 dBfs. Quarter power is -6 dBfs. Zero power is -infinity dBfs.